IdentityCore/src/util/ios/MSIDKeychainUtil.m

```diff
@@ -98,11 +98,7 @@ - (NSString *)appDefaultAccessGroup
             || readStatus == errSecInteractionNotAllowed)
         {
             NSMutableDictionary* addQuery = [query mutableCopy];
-#if TARGET_OS_MACCATALYST
```

Comment:
Hmmm, not sure this change is correct. Seems previous condition was kSecAttrAccessibleAfterFirstUnlock on catalyst, and kSecAttrAccessibleAlways everywhere else.

Reply:
reverted and added pragma for suppressing warning about deprecated api